### PR TITLE
Added option to start with the mouse hidden

### PR DESCRIPTION
**Pull request by sqrtroot**
Small new feature, I needed this for a project.
The mouse starts hidden, instead of showing. Removing the need to wait at least one timeout period. The option is behind a CLI flag as to not change the default behaviour.

### CODE REVIEW

**Comment by sqrtroot** [comment]
Build seems to fail on making the docs? Building docs works locally.

**Review by Airblader — CHANGES_REQUESTED** [review]
Thanks for the PR! I left a few comments, would be great if you could address those.

File: src/unclutter.c [hunk 1]
```diff
@@ -231,4 +236,6 @@ static void display_init() {
     active_root = RootWindow(display, active_screen);
     default_screen = active_screen;
     cursor_find(&child, &root_x, &root_y);
+    if(config.start_hidden)
```
[the remark this answers is not in the document]
Comment (sqrtroot) [reply src/unclutter.c:239]: Ah I think the name confused me, I've moved it to run as that seems the next logical place to me

**Review by Airblader — CHANGES_REQUESTED** [review]
Thanks! The code looks good now. Can you please squash the commits into one and force-push again? Then just drop a comment and we're good to go.

**Comment by sqrtroot** [comment]
Any progress on this?

**Review by Airblader — APPROVED** [review]
Sorry, I don't get notifications for pushes, only comments. All good now, thank you!

**Pull request merged**